distribution/openrct2.d.ts

```diff
@@ -860,13 +860,19 @@ declare global {
         readonly shopItemSecondary: number;
     }
 
+    /**
+     * Represents a VehicleSpriteGroup
+     */
+     interface SpriteGroup {
+         readonly imageId: number;
+         readonly spriteNumImages: number;
+     }
```

Comment:
How would plugin developers be able differentiate between different sprite groups?